sharpd/sharp_vty.c

```diff
@@ -81,46 +82,63 @@ DEFPY(watch_nexthop_v4, watch_nexthop_v4_cmd,
 
 DEFPY (install_routes,
        install_routes_cmd,
-       "sharp install routes A.B.C.D$start nexthop <A.B.C.D$nexthop4|X:X::X:X$nexthop6> (1-1000000)$routes [instance (0-255)$instance]",
+       "sharp install routes A.B.C.D$start [nexthop <A.B.C.D$nexthop4|X:X::X:X$nexthop6>|nexthop-group NAME$nexthop_group] (1-1000000)$routes [instance (0-255)$instance]",
```

Comment:
Shouldn't we use <> instead of [] so that either `nexthop` or `nexthop-group` needs to be specified?

Reply:
yeah my mistake. Fixed